go/engine/pgp_pull.go

```diff
@@ -128,20 +127,71 @@ func (e *PGPPullEngine) getTrackedUserSummaries(ctx *Context) ([]keybase1.UserSu
 	return matchedList, nil
 }
 
-func (e *PGPPullEngine) Run(ctx *Context) error {
-	summaries, err := e.getTrackedUserSummaries(ctx)
+func (e *PGPPullEngine) runLoggedOut(ctx *Context) error {
+	if len(e.userAsserts) == 0 {
```

Comment:
Pull for all trackers.

On Thursday, November 5, 2015, oconnor663 <notifications@github.com> wrote:

> In go/engine/pgp_pull.go
> <https://github.com/keybase/client/pull/1289#discussion_r44101595>:
>
> > @@ -128,20 +127,71 @@ func (e *PGPPullEngine) getTrackedUserSummaries(ctx *Context) ([]keybase1.UserSu
> >  	return matchedList, nil
> >  }
> >
> > -func (e *PGPPullEngine) Run(ctx *Context) error {
> > -	summaries, err := e.getTrackedUserSummaries(ctx)
> > +func (e *PGPPullEngine) runLoggedOut(ctx *Context) error {
> > +	if len(e.userAsserts) == 0 {
>
> Does this not also get checked in the logged in version? What does it mean
> to pull a PGP key with no user?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1289/files#r44101595>.
>